include/version.h

```diff
@@ -22,3 +25,4 @@
 #define APP_FULLNAME  PACKAGE "/" VERSION
 #endif
 
+#endif /* SQUID_VERSION_H */
```

Comment:
git complains, missing newline before EOF
```suggestion
#endif /* SQUID_VERSION_H */

```